components/admin_console/license_settings/modals/upload_license_modal.test.tsx

```diff
@@ -58,17 +59,17 @@ describe('components/admin_console/license_settings/modals/upload_license_modal'
 
     const props = {
         onExited: mockOnExited,
-        fileObjFromProps: {} as File,
+        fileObjFromProps: {name: 'Test license file'} as File,
     };
 
     const mockStore = configureStore([thunk]);
     const store = mockStore(state);
 
     test('should match snapshot when is not licensed', () => {
         const wrapper = shallow(
-            <Provider store={store}>
+            <reactRedux.Provider store={store}>
```

Comment:
nit: out of curiosity, why did you need to use the provider this way?

Reply:
I make a call to mock one of the functions from react-redux 
```js
const useDispatchMock = jest.spyOn(reactRedux, 'useDispatch');
```
That's how I ended doing it that way